cmd/oras/internal/option/remote.go

```diff
@@ -49,12 +49,14 @@ import (
 // Remote implements oerrors.Handler and interface.
 type Remote struct {
 	DistributionSpec
-	CACertFilePath    string
-	Insecure          bool
-	Configs           []string
-	Username          string
-	PasswordFromStdin bool
-	Password          string
+	CACertFilePath         string
+	Insecure               bool
+	Configs                []string
+	Username               string
+	PasswordFromStdin      bool
+	Password               string
+	IdentityTokenFromStdin bool
```

Comment:
All read from stdin should be conflict with `xxxFromStdin`:

`oras manifest push` and `oras manifest push` accepts `-` to read content from stdin, should be conflict with both `--IdentityTokenFromStdin` and `--PasswordFromStdin`

Reply:
To be more specific, checks both `--IdentityTokenFromStdin` and `--PasswordFromStdin` when file reference is set to `-` and wrap the check function in `option` package.

Reply:
added a check function in the `option` package.